lets-encrypt_acme_renew_script/renew.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#Author: TechnoMan, mail: git@frezbo.com
+document_root=/var/www/html # change to your document root
+scripts_directory=/usr/local/scripts # scripts directory
+acme_directory=acme-tiny/lets-encrypt_acme_renew_script
+nginx_config_location=/etc/nginx #change if this is not your config directory
+#certificate file location
+certs_location=${nginx_config_location}/ssl/certs
+#domain keys location
+keys_location=${nginx_config_location}/ssl/private
+#file containing list of domains
+domain_file=${scripts_directory}/${acme_directory}/domains.txt
+#file containing CSR genreation parameters
+csr_params=${scripts_directory}/${acme_directory}/domains.cnf
+#intermediate certificate
+int_cert=${scripts_directory}/${acme_directory}/intermediate.pem
+#current unix time
+curr_time=$(date +%s)
+touch .rnd
+export RANDFILE=${scripts_directory}/${acme_directory}/.rnd
+#check whether certifcate expires within 30 days
+for i in $(cat $domain_file)
+do
+#certifcate expiry unix time
+((cert_expire_time=$(date -d "$(openssl x509 -in ${certs_location}/$i.crt -noout -enddate | sed s/notAfter=//g)" +%s)))
+#days to expire
+((days_to_expire=(cert_expire_time-curr_time)/86400))
+if [ $days_to_expire -le 30 ]
+then
+echo "$days_to_expire days untill expiry"
+openssl genrsa -out ${scripts_directory}/${acme_directory}/$i.key 4096
+chmod 600 ${scripts_directory}/${acme_directory}/$i.key
+openssl ecparam -genkey -name secp384r1 | openssl ec -out ${scripts_directory}/${acme_directory}/$i.ec.key
+chmod 600 ${scripts_directory}/${acme_directory}/$i.ec.key
+sed -i "8!s/example.com/$i/g" $csr_params
```

Comment:
I would recommend some other tag, like `__DOMAIN__`. This is more clear and less error prone.